components/Download.vue

```diff
@@ -99,6 +99,10 @@ export default class extends Vue {
 
     const packageid = generatePackageId((this.manifest.short_name as string) || (this.manifest.name as string));
 
+    let startURL = (this.manifest.start_url as string).replace(`https://${new URL(this.siteHref).hostname}`, "")
+    console.log(startURL);
```

Comment:
I know you and David did some work to remove console logs recently - does it matter that this one is here?